app/controllers/manifest_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ManifestController < ApplicationController
+  def show # rubocop:disable Metrics/MethodLength
+    render json: {
+      short_name:       AppConfig.settings.pod_name,
+      name:             AppConfig.settings.pod_name,
+      description:      "diaspora* is a free, decentralized and privacy respectful social network",
```

Comment:
It's a bit late now, but I would suggest 'privacy-respecting' rather than 'privacy respectful'.

Reply:
It's never to late, I just changed that now in 035b6f39fcc91f5f18c1d2cc6357d1f9a37001fd :+1:

Reply:
Thanks (thumbs up).